mypy/build.py

```diff
@@ -2398,12 +2398,38 @@ def skipping_ancestor(manager: BuildManager, id: str, path: str, ancestor_for: '
                           severity='note', only_once=True)
 
 
+def log_configuration(manager: BuildManager) -> None:
+    """Output useful configuration information to LOG and TRACE"""
+
+    manager.log()
+    manager.log("Mypy version %s" % __version__)
```

Comment:
I'd recommend aligning the various fields -- we do that also in `dump_stats()` near line 561.